test/serializers/associations_test.rb

```diff
@@ -126,6 +126,35 @@ def test_associations_custom_keys
         assert expected_association_keys.include? :site
       end
 
+      class InlineAssociationTestPostSerializer < ActiveModel::Serializer
+        has_many :comments
+        has_many :last_comments do
+          object.comments.last(1)
```

Comment:
This is a great example @beauby made of why you'd want to define your own association value

However, the way that this works, it's really the same as `attribute :last_comments do end`, right?  Unless this affects deserialization, right?